wire-ios/Wire-iOS/Sources/UserInterface/Settings/Backup/BackupStatusCell.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+final class BackupStatusCell: UITableViewCell {
+
+    let descriptionLabel: DynamicFontLabel = {
+        let label = DynamicFontLabel(fontSpec: .normalRegularFont,
+                                     color: SemanticColors.Label.textDefault)
```

Comment:
suggestion: Since you're already on it, can you try using the new initializer and make sure it looks good with a bigger font size?